src/sql/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -337,6 +337,24 @@ class WelcomePage extends Disposable {
 		this.createDropDown();
 		this.createWidePreviewToolTip();
 		this.createPreviewModal();
+		this.handleAccessibility(this.commandService);
+	}
+
+	private async handleAccessibility(commandService: ICommandService): Promise<void> {
+		const tileServer = document.querySelector('#tile-server-link') as HTMLElement;
+
+		addStandardDisposableListener(tileServer, 'keydown', event => {
+			if (event.equals(KeyCode.Enter)) {
+				const historyLabel = document.querySelector('#historyLabel') as HTMLElement;
+				commandService.executeCommand('azdata.resource.deploy');
```

Comment:
you can extract this to a method and reuse it down below.

Reply:
@alanrenmsft fixed

Reply:
what i meant was extract the following code into a method, and you can call it here and below.
`
const historyLabel = document.getElementById('historyLabel') as HTMLElement;
					commandService.executeCommand('azdata.resource.deploy');
					historyLabel.focus();
`